### DISCUSSION TITLE
Module 3 Exercise 1

### DISCUSSION THREAD

**Discussion by reygyimah** [SERVIR-WA/GALUP#299] [Q&A]
I am still unable to get my images. That of elevation is the only one that was processed. 
LST appear dark and the rest doesn't show at all.
Please find attached for your assistance.
Thanks
<img width="1440" alt="Screenshot 2021-11-23 at 1 10 56 AM" src="https://user-images.githubusercontent.com/93355003/142957829-eac1d443-68c7-40da-854c-c3c7e0b5209c.png">

**Comment by JuliePeeling** [comment]
Hi Ronald,

One suggestion is to try using Landsat 7 (IMG_L7) instead of Landsat 5 (IMG_L5). This data collection may have more images available during your time period within the area of interest. Else, it may help to try a different area that is less prone to cloud cover. Please let us know if these methods are not sufficient, and we will continue troubleshooting.

Best,
Julie

**Comment by reygyimah** [comment]
Thank for the prompt reply. How do I change to landsat 7. I was thinking
all the images for landsat 5 7 and 8 will be processed at a goal.

**Reply by JuliePeeling** [reply]
You are correct in that there is a single function to process all three data collections, but to specify the collection in the code, you must apply the function to a specific collection. In the script, around line 119, the code contains this snippet: "var IMG_LT = getLSAT(IMG_L8, REG_GH, ST_DATE, EN_DATE);" . In this example, Landsat 8 (IMG_L8) is used, and in your code, Landsat 5 (IMG_L5) is most likely used, but if you change this to IMG_L7, that should give you Landsat 7. Let me know if further clarification is needed.
Best,
Julie

**Comment by reygyimah** [comment]
Hello this is the new image and errors now after I change to Landsat 7 and location as directed.
I now have the Landsat, NDVI and elevation image but LST appears whitish.

I count on your usual cooperation.

**Comment by JuliePeeling** [comment]
Hi Ronald, 
I am unable to see the images that you mentioned, but the discoloration of LST may simply be due to the visualization parameters. LST is programmed to appear in black and white, so if it is only appearing white, try adjusting the visualization parameters until you are satisfied with the representation. If there are still issues and you would like to include a screenshot of your GEE map, please feel free to do so.

Best,
Julie
